the-devastating-chickens/BangazonSite
# Av product details

# Description

Product Details will show the quantity remaining (total qty - number of orders containing that product). The product details are also accessible from the list of products that appear in product typesdetails.

Fixes Issue #2

## Type of change


# Testing Instructions

```git fetch --all```
```git checkout av-product-details```
Start program
Navigate to ProductTypes
Click on product type details
Click on the title of a product
Confirm that the Quantity Remaining property is less than the quantity property that appears in the list view.
Alternatively, you can type localhost5000/products to see a list of all products and view details there. 


# Checklist:
